pages/dashboard.js

```diff
@@ -2,25 +2,43 @@ import { useState, useEffect } from 'react';
 import styled from 'styled-components';
 import { useContext } from 'react';
 import useUser from '../hooks/useUser';
-import { accountByEmail } from '../graphql/api';
+import { accountByEmail, updateInterviewNotes } from '../graphql/api';
 import Context from '../store/context';
 import ApplicationList from '../components/dashboard/applicationList';
 import InterviewList from '../components/dashboard/interviewList';
 import InterviewOverview from '../components/dashboard/interviewOverview';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
 const Content = styled.div`
   min-height: 100vh;
   width: 100%;
 `;
 
+toast.configure();
 export default function Dashboard() {
   const user = useUser({ redirectTo: '/signup', redirectIfFound: false });
   const [profile, setProfile] = useState(null);
   const [email, setEmail] = useState(null);
   const [applications, setApplications] = useState([]);
-  const [errorMessage, setErrorMessage] = useState(null);
+  const [errorMessage, setErrorMessage] = useState(null);  
+  const [notes, setNotes] = useState('');
   const { state } = useContext(Context);
 
+  const onChange = (e) => {
+    setNotes(e.target.value);
+  };
+
+  const saveInterviewNotes = async (id, date, type) => {
+    const { data, errorMessage } = await updateInterviewNotes(id, notes, date, type);
+    if (data?.updateInterview.notes === notes) {
+      toast.success("Notes saved! 🚀");
+    }
+    if(!!errorMessage) {
```

Comment:
there's nothing wrong with your syntax here, however it is not very common in JS. JS uses type coercion when a value is in a boolean context like this one. Basically it automatically transforms the value to a boolean, so we dont really have a need to prefix a value with `!!`. In this case you can just do `if(errorMessage)`. More info [here](https://developer.mozilla.org/en-US/docs/Glossary/Truthy).

Reply:
Yeah I'm aware haha, I just find `!!` easier to interpret most of the time (though admittedly i'm not too consistent with it). I don't mind changing this though if you prefer